lib/server.js

```diff
@@ -313,12 +317,31 @@ class NodeServer extends ServerBase {
     this.closed = false
     this.server.on('connection', this.onConnection.bind(this))
     this.server.on('request', this.wrapRequest.bind(this))
+    if (this.client.natTraversal != null && this.nat === true) {
+      this.server.on('listening', () => {
+        try {
+          const port = this.server.address().port
```

Comment:
this wont actually work, as at this time the port isnt defined, its defined a few lines later on the this._listen.apply

Reply:
Thanks for spotting this, as per the above comment, NAT has been removed for the HTTP server.